receiver/RatingReceiver.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+import websockets
+import json
+
+async def listen_for_new_ratings():
+    async with websockets.connect("ws://localhost:8765") as websocket:
+        while True:
+            rating_data = json.loads(await websocket.recv())
+
+            print(f"Rating data has been received: {rating_data}")
```

Comment:
Let's go a step further and start using `logger.info` instead of `print`.
In order to do that read up a bit on pythons default logger library.

Reply:
On it

Reply:
Done.